README.md

```diff
@@ -98,124 +91,103 @@ oc edit clusterrole admin # add the above and save
 oc edit clusterrole edit # add the above and save
 ```
 
-### Setup the plugin
+### Setup as Kubectl/OC plugin
 
 - have the [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) command line tool or the [oc command line tool](https://docs.openshift.org/latest/cli_reference/get_started_cli.html#installing-the-cli) already installed
 - it should be version k8s version 1.8 or OpenShift 3.7 or later
 - create a new dir ```mkdir -p ~/.kube/plugins/mobile```  
 - copy the cli_plugin.yaml file to the dir ```cp ./artifacts/cli_plugin.yaml ~/.kube/plugins/mobile```
-- rename the ```cli_plugin.yaml``` file to ```plugin.yaml```
+and rename the ```cli_plugin.yaml``` file to ```plugin.yaml```
 - install the mobile CLI binary onto your $PATH
 
-### Basic usage
+## Basic usage
 
-To access the plugin you can use the following command:
-```kubectl plugin mobile <command>```
-
-You can also use it standalone as it will pick up on your kube configuration:
-```mobile --namespace=mine <command>``` notice we pass the namespace here.
+To use the mobile CLI as a plugin you can use the following command:
+``` bash
+kubectl plugin mobile <command>
+oc plugin mobile <command>
+```
 
 **Passing flags**
 
-To pass flags when using the plugin ensure to use the ```--``` option:
-```kubectl plugin mobile -- create client --<someflag> ```
+To pass flags when using the mobile CLI as a plugin, ensure to use the ```--``` option:
+```bash
+kubectl plugin mobile create client -- --<someflag>
+oc plugin mobile create client -- --<someflag>
+```
+
+The mobile CLI can also be use standalone as it will pick up on your kube configuration:
+```bash
+mobile <command> --namespace=mine 
+``` 
 
+**NOTE: When the mobile CLI is used as an oc plugin, the namespace does not have to be specified as this
```

Comment:
Maybe this could be shortened to:
**NOTE: When this CLI is used as an OC plugin, you do not need to provide the `--namespace` flag.**

What do you think?